goosebit/db/__init__.py

```diff
@@ -5,6 +5,7 @@
 
 async def init():
     await Tortoise.init(config=TORTOISE_CONF)
+    await Tortoise.generate_schemas()
```

Comment:
Should not be necessary. As written in the README.md : "Create or upgrade database: poetry run aerich upgrade"

My working assumption is, that DB manipulations are best left to the Dev / Ops person. Don't automate it within the app.